website/blog/2021/01.25-Machine-Controller-Manager.md

```diff
@@ -95,7 +95,7 @@ Metal-stack is a set of microservices that implements Metal as a Service (MaaS).
 
 Sky UK Limited (a broadcaster) migrated their Kubernetes node management from Ansible to Machine Controller Manager. Check out the [How Sky is using Machine Controller Manager (MCM) and autoscaler](https://youtu.be/yF4wq7GAeEM) video on our [Gardener Project](https://www.youtube.com/channel/UCwUhwKFREV8Su0gwAJQX7tw) YouTube channel.
 
-Also, other interesting use cases with MCM are implemented by Kubernetes enthusiasts, who for example adjusted the Machine Controller Manager to provision machines in the cloud to extend a local Raspberry-Pi K3s cluster. Read more about it in [Scale out your Raspberry-Pi Kubernetes cluster to the cloud](https://nativecloud.dev/scale-out-your-raspberry-pi-k3s-cluster-to-the-cloud/) or check out [this video](https://youtu.be/UuveyEOn4_o?t=60) on our [Gardener Project](https://www.youtube.com/channel/UCwUhwKFREV8Su0gwAJQX7tw) YouTube channel.
+Also, other interesting use cases with MCM are implemented by Kubernetes enthusiasts, who for example adjusted the Machine Controller Manager to provision machines in the cloud to extend a local Raspberry-Pi K3s cluster. For more information, you can check out [this video](https://youtu.be/UuveyEOn4_o?t=60) on our [Gardener Project](https://www.youtube.com/channel/UCwUhwKFREV8Su0gwAJQX7tw) YouTube channel.
```

Comment:
Shall we replace "this video" with the video name, in this case "Gardener Community Meeting 2020-07-03"?